google-cloud-speech/lib/google/cloud/speech.rb

```diff
@@ -35,6 +35,13 @@ module Cloud
     # 3. [Enable the Google Cloud Speech API.](https://console.cloud.google.com/apis/api/speech)
     # 4. [Setup Authentication.](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/master/guides/authentication)
     #
+    # ### Migration Guide
+    #
+    # This release, 0.30.0, introduces breaking changes relative to the previous
```

Comment:
I would recommend future-proofing this sentence with: "The 0.30.0 release introduced ..."